concent_api/core/message_handlers.py

```diff
@@ -721,6 +727,12 @@ def handle_send_force_payment(
         subtask_results_accepted.payment_ts for subtask_results_accepted in client_message.subtask_results_accepted_list
     )
 
+    for subtask_results_accepted in client_message.subtask_results_accepted_list:
```

Comment:
There is no changes being done on those `Subtasks`, so locking them in this particular use case is not necessary.

Reply:
Right.. I checked it again and I see only one possible problem- without locking it might be danger that other process will change something in this subtask during this flow.. And we are not changing anything is subtask, but we are storing pending_response with values basing on values in subtask.. If it is real problem and this problem is even possible- I'm really not sure. On the other hand, generally even if it is not necessary it 'costed' us not much and might prevent data corruption.. 
What do you think? I'm not sure.. Should I leave it or remove?

Reply:
This handler is not supposed to even look at `Subtask` objects in the database. We never store anything submitted in this use case. Note that there's no corresponding state in the state machine. The pending response table has a special case for responses created here so that they do not need to refer to a subtask.

So @rwrzesien is right. This is not necessary and should be removed.

If you find anything in this use case that makes a query to the `Subtask` table, please report it as a bug.

Reply:
The `store_pending_message` in this case do not use anything from `Subtask` table, only from client message. Please check the call of it - `subtask` parameter is skipped there on purpose, because there can be many. So I think there is no need for locking this one, unless there is a reason I can't see now :)

Reply:
done, thanks for good explanation.